app/assets/javascripts/comments.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
it's good practice to not include empty files to commits. If some file was autogenerated and has no code - you can just removed it